usl_pipeline/usl_pipeline/formats/geotiff_loader.py

```diff
@@ -0,0 +1,39 @@
+import rasterio.features
+
+
+def load_elevation_from_geotiff(
+    file_path: str, load_data: bool = False, result_nodata: float = None
+):
+    """
+    Loading elevation raster data from GeoTIFF file.
+    :param file_path: path to a file to load from
+    :param load_data: indicates that in addition to header, data should also be
+            loaded
+    :param result_nodata: optional new value for special case of no data in
```

Comment:
I'd name & describe this as something like

:param no_data_value: Value to set in the returned data to indicate absence of data. If not supplied, the no-data value defined in the tiff file itself will be used.

Reply:
Done